src/discussions/comments/data/selectors.js

```diff
@@ -1,10 +1,10 @@
 /* eslint-disable import/prefer-default-export */
-export const selectThreadComments = threadId => state => (state.comments.threadCommentMap[threadId] || []).map(
-  commentId => state.comments.comments[commentId],
+export const selectThreadComments = threadId => state => (state.comments.commentsInThreads[threadId] || []).map(
+  commentId => state.comments.commentsById[commentId],
 );
 
-export const selectCommentResponses = commentId => state => (state.comments.commentResponsesMap[commentId] || []).map(
-  cId => state.comments.comments[cId],
+export const selectCommentResponses = commentId => state => (state.comments.commentsInComments[commentId] || []).map(
+  cId => state.comments.commentsById[cId],
 );
```

Comment:
I am a fan of reselet so no issues using it. I just avoid introducing new libraries that aren't already used. If this is OK, I can definitely clean up the seletors with it.

Reply:
I think reselect is available in the redux toolkit so you won't have to add another library for that.

Reply:
I've updated the code to use reselect. That said we currently can't take advantage of most of its feature related to memorization since we'll be creating selectors dynamically.